DFE-Digital/Dfe.Academies.External
# invite a contributor - remove dupe role col header text

See below ticket:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/113672?McasTsid=26110

## Type of change

## Expected behaviour


## Steps to reproduce issue (if relevant)
1. go to application overview
2. click on contributors link
3. see 'role' text above 'remove contributor' links

## Steps to test this PR
1. go to application overview
2. click on contributors link
3. see '' text above 'remove contributor' links

## Prerequisites
n/a
